test/mpi/pt2pt/testlist.in

```diff
@@ -64,3 +64,7 @@ precv_anysrc 2 timeLimit=60
 precv_anysrc 2 timeLimit=60 env=MPIR_CVAR_CH4_OFI_AM_LONG_FORCE_PIPELINE=true
 precv_anysrc_exp 2 timeLimit=60
 pt2pt_large 2
+mpit_isendirecv 2 arg=0 arg=0
+mpit_isendirecv 2 arg=0 arg=1
+mpit_isendirecv 2 arg=1 arg=0
+mpit_isendirecv 2 arg=1 arg=1
```

Comment:
minor nit, but does this test belong more in `test/mpi/mpi_t`?

Reply:
I think it went in pt2pt because it actually did communication while the mpi_t directory didn't have anything like that.

Reply:
Hm, I'd still rather these live in the tools dir, since that's what we're really testing here. At least one test in there makes a communication call, so I don't think they'd be out of place.

Reply:
Ok. I'll move them then.

Reply:
Done.